02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2036 @@
+{
```

Comment:
<p>I'm not sure what the phrase "fully qualified list of levels" means here. Is this just to say "the full list of levels"</p><p> </p><p>Edit: I realize you mean "fully qualified" in the technical sense, but I think you only briefly define it above. It might deserve a bit more robustness if you're going to use it quite a bit. At which point it would suggest explaining why someone would want the fully qualified list.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>This is a great point, when I was mentioning the "fully qualified list of levels" in the <code>.levels</code> section, it was not with the same intent as when I was talking about the "fully qualified one (with both the dimension and hierarchy listed)" for the <code>.name</code> section. Therefore, I've amended the <code>.levels</code> section to be the following:</p><p> </p><p><strong>But let's say we wanted to see just the levels for our previous multi-level hierarchy&nbsp;</strong><code><strong>Product</strong></code><strong>. Scanning through the full list of levels in the cube may be a bit cumbersome (especially if we have a large amount of levels!). Thus, we can instead use the&nbsp;</strong><code><strong>levels</strong></code><strong>&nbsp;property for the multi-level&nbsp;</strong><code><strong>Product</strong></code><strong>&nbsp;hierarchy object to only show the levels for that particular hierarchy.</strong></p>